root/artist/edit_form.tt

```diff
@@ -97,16 +101,93 @@
   </div>
 
   <div class="documentation">
-    [%- area_bubble() -%]
+    <div class="bubble" id="name-bubble">
+      <p>
+        [% l('The {doc|name} is the artist’s official name.',
+             { doc => { href => doc_link('Artist#Name'), target => '_blank' } }) %]
+      </p>
+      <p>
+        [% l('Please see the {doc|style guidelines} for more information.',
+             { doc => { href => doc_link('Style/Artist#Name'), target => '_blank' } }) %]
+      </p>
+    </div>
 
     [%- sort_name_bubble() -%]
 
+    <div class="bubble" id="comment-bubble">
+      <p>
+        [% l('The {doc|disambiguation} field is used to help users distinguish between identically‐ or similarly‐named artists.
+              It’s okay to leave it blank if it seems unlikely that there will be other artists with similar names.',
```

Comment:
I'm probably revealing my pro-disambiguation bias here, but please let me know how this sounds. :-)

Reply:
I think for artists being pro-disambiguation makes sense.

"between identically‐ or similarly‐named artists" - "between artists with the same or similar names" maybe? I mean, I know you love your hyphens but it might be easier to understand for people with worse English :)

Reply:
Thanks, I prefer that too. If it works for you, I'm going with "... with identical or similar names" since I think it reads a bit more clearly.

Reply:
I know this is very late, and I didn't pick it up on the other PR which uses the same language - but I think the phrasing could be a bit more 'active':
"The disambiguation field helps users distinguish between artists..."
_or even_
"The disambiguation field distinguishes between artists..."
_instead of_
"The disambiguation field is used to help users distinguish between artists..."

But I am okay with leaving it, to keep it consistent and to avoid complicating things :)

P.P.S Maybe you are all more used to working in github, but next time can we lay it all out in a text doc first :P

Reply:
Thanks for the suggestion! I think I prefer the "field helps users distinguish" version, and I can probably change the artist string in this change too. @reosarevok?

Re using a text doc, do you mean creating a shared Google doc to work out all the strings or something like that?